next/public/locales/sk/forms.json

```diff
@@ -7,6 +7,8 @@
   },
   "selectAll": "Vybrať všetko",
   "deselectAll": "Odobrať všetko",
+  "options1": "Možnosti",
+  "options2": "Možností",
```

Comment:
would use the _zero _one _few _many suffix (and don't forget to handle the grammar for 0 and 1)

https://www.i18next.com/translation-function/plurals

Reply:
zero and one are not needded now but i can create these also